wallet/src/de/schildbach/wallet/WalletApplication.java

```diff
@@ -671,28 +674,25 @@ public void lockWalletIfNeeded() {
     }
 
     /**
-      Replace the wallet with an new wallet as part of a wallet wipe
+     * Removes all the data and restarts the app showing onboarding screen.
      */
-    public void eraseAndCreateNewWallet() {
-        Wallet newWallet = Wallet.createDeterministic(Constants.NETWORK_PARAMETERS, Script.ScriptType.P2PKH);
-
-        newWallet.addKeyChain(Constants.BIP44_PATH);
+    public void wipe(Context context) {
+        log.info("Removing all the data and restarting the app.");
 
-        log.info("creating new wallet after wallet wipe");
+        resetBlockchain();
+        wallet.shutdownAutosaveAndWait();
+        stopBlockchainService();
 
-        File walletBackupFile = getFileStreamPath(Constants.Files.WALLET_KEY_BACKUP_PROTOBUF);
-        if(walletBackupFile.exists())
-            walletBackupFile.delete();
```

Comment:
Why is the code to remove the backup file removed?

Reply:
Though the backup file would be recreated and therefore replace the old one during the onboarding process.  But we should check on that.

Reply:
I removed this code, since noticed it is duplicated by what is happening in `cleanupFiles()` method:
https://github.com/dashevo/dash-wallet/pull/266/files/1de855c9542b4460f5aa344238a3adc099ae9aa1#diff-cbdcd0a31a45f4288739e96a6627f2faR526